packages/react-charting/bundle-size/VerticalBarChart.fixture.js

```diff
@@ -0,0 +1,9 @@
+import { VerticalBarChart } from '@fluentui/react-charting';
+
```

Comment:
can we have a single file for all charts

Reply:
Ok

Reply:
![image](https://github.com/microsoft/fluentui/assets/35366351/46cb3a6e-9b60-46aa-bad6-7a6fbb16ae7f)
FluentUI react has separate file for each component